### PR TITLE
fix(button): set vertical alignment for md-button and md-raised-button

### PR DESCRIPTION
**Pull request by tinayuangao**
To make the md-icon correctly aligned

For issue #1454 

R: @jelbourn

### CODE REVIEW

File: src/lib/button/button.scss [hunk 1]
```diff
@@ -47,6 +47,12 @@
   }
 }
 
+[md-button], [md-raised-button] {
```
Comment (jelbourn) [inline src/lib/button/button.scss]: Could you add a comment that explains what this style is for / why it was added?

Comment (tinayuangao) [inline src/lib/button/button.scss]: Done.

**Comment by jelbourn** [comment]
LGTM

**Comment by helzgate** [comment]
Did anyone find that this actually aligned the icon?  As far as I can tell it didn't fix the issue.

**Comment by helzgate** [comment]
Guess I'm not loading something correctly, what I have no idea.  I manually added 
`[md-button], [md-raised-button] {
   .md-button-wrapper > * {
     vertical-align: middle;
   }
 }` 
to my main sass file to fix the issue.

**Comment by afriedma** [comment]
Can you guys please assist, do you know how to align text right below the icon?

**Pull request merged**